stubs/_blueman.pyi

```diff
@@ -0,0 +1,58 @@
+from typing import Any, List, Dict, Union, Callable
+from mypy_extensions import TypedDict
+
+ERR: Dict[int, str]
+RFCOMM_HANGUP_NOW: int
+RFCOMM_RELEASE_ONHUP: int
+RFCOMM_REUSE_DLC: int
+RFCOMM_STATES: List[str]
+RFCOMM_TTY_ATTACHED: int
+
+class RfcommDev(TypedDict):
```

Comment:
Just a side note: If we define this here, mypy thinks it exists in the module and would accept code like `from _blueman import RfcommDev` although it will not actually work. Not sure if there's a way to avoid that, though.

Reply:
I'll check if I can hide it somehow, until then I can prefix it with and underscore.